modules/rosa-adding-instance-types.adoc

```diff
@@ -0,0 +1,50 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-managing-worker-nodes.adoc
+
+[id="rosa-adding-instance-types_{context}"]
+= Adding instance types
+
+[role="_abstract"]
+After a machine pool is created, the instance type cannot be changed. To add a different instance type for worker nodes, you must create a new machine pool for the additional instance type.
+
+.Procedure
+
+. To add an instance type with a new machine pool, enter the following command:
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=<cluster-name> <machinepool_ID> --instance-type=<type>
+----
++
+.Example input
```

Comment:
Our convention is to use the example header right above the code block as you do later in line 44. Listing it ahead of a sentence would break that, so this should be:
```
This example creates a new machine pool with the instance type `m5.2x.large`, 2 replicas, and labels on a cluster named `mycluster`:
+
**.Example output**
[source,terminal]
----
$ rosa create machinepool --cluster=mycluster db-nodes-large-mp --replicas=2 --labels=app=db,tier=backend --instance-type=m5.2xlarge
----
```

Reply:
Fixed. Thank you  - I miss having a preview to check this with.